src/components/BeamContext.tsx

```diff
@@ -7,9 +7,10 @@ import { SnackbarProvider } from "src/components/Snackbar/SnackbarContext";
 import { SuperDrawer } from "src/components/SuperDrawer/SuperDrawer";
 import { ContentStack } from "src/components/SuperDrawer/useSuperDrawer";
 import { CanCloseCheck, CheckFn } from "src/types";
-import { EmptyRef } from "src/utils/index";
+import { EmptyRef, isDefined } from "src/utils/index";
 import { RightPaneProvider } from "./Layout";
 import { ToastProvider } from "./Toast/ToastContext";
+import { ModalContext, ModalProvider } from "./Modal/ModalContext";
```

Comment:
**request:** Can you remove these unused imports? I may have accidentally left them in when I made changes.